Applications/Terminal/TerminalView.m

```diff
@@ -2967,6 +2967,15 @@ - (void)setAlternateAsMeta:(BOOL)altAsMeta
 // ---
 // Title (window, icon)
 // ---
+- (NSString *)windowTitle
+{
+  return title_window;
+}
+- (NSString *)miniWindowTitle
+{
+  return title_miniwindow;
+}
+
```

Comment:
This method is unused. Moreover I think miniwindow title should be constructed dirreferently then window title and its logic should be placed inside TerminalWindow class.